eo-parser/src/main/antlr4/org/eolang/parser/Program.g4

```diff
@@ -171,10 +171,13 @@ head
     HOME
     |
     AT
+    DOT?
     |
     RHO
+    DOT?
     |
     XI
+    DOT?
     |
     SIGMA
```

Comment:
@Graur why not here too? I think that it's better to group them:

```
(
AT
|
RHO
|
XI
|
SIGMA
)
DOT?
```

Reply:
@yegor256 Good idea, thanks!